protocol/x/clob/abci.go

```diff
@@ -174,45 +174,6 @@ func PrepareCheckState(
 		offchainUpdates,
 	)
 
-	// For orders that are filled in the last block, send an orderbook update to the grpc streams.
```

Comment:
Axed this whole section to send out order updates.
Reason being:
- PCS check state is set to deliver state, so no additional updates are needed.
- since fills are directly emitted immediately on orderbook update, matched orders don't need to be updated